app/javascript/components/host-initiator-group-form/index.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import MiqFormRenderer from '@@ddf';
+import { Loading } from 'carbon-components-react';
+import createSchema from './host-initiator-group-form.schema';
+import miqRedirectBack from '../../helpers/miq-redirect-back';
+
+const HostInitiatorGroupForm = ({ redirect, storageManagerId }) => {
+  const [state, setState] = useState({});
+  const [storageId, setStorageId] = useState(undefined);
+  const { isLoading, initialValues } = state;
+
+  const loadSchema = (appendState = {}) => () => {
+    setState((state) => ({
+      ...state,
+      ...appendState,
+    }));
+  };
+
+  useEffect(() => {
+    if (storageManagerId) {
+      API.options(`/api/host_initiator_groups?ems_id=${storageManagerId}`)
+        .then(loadSchema({ initialValues: { ems_id: storageManagerId }, isLoading: false }));
+    }
+  }, [storageManagerId]);
+
+  const onSubmit = async(values) => {
+    miqSparkleOn();
+    const message = __('Host initiator group define');
```

Comment:
Could you change message to  ` Add of Host initiator group "%s" has been successfully queued.`? %s is values.name. For examples you can check in other forms.